stories/Dropdown/examples.jsx

```diff
@@ -22,17 +22,33 @@ class Controlled extends React.Component {
         selectedItem={selectedItem}
         onChange={(item) => this.setState({ selectedItem: item })}
         items={[
-          { label: 'Amazonas', value: 'AM' },
-          { label: 'Bahia', value: 'BA' },
-          { label: 'Ceará', value: 'CE' },
-          { label: 'Goiás', value: 'GO' },
-          { label: 'Minas Gerais', value: 'MG' },
-          { label: 'Paraná', value: 'PR' },
-          { label: 'Pernambuco', value: 'PE' },
-          { label: 'Rio de Janeiro', value: 'RJ' },
-          { label: 'Rio Grande do Sul', value: 'RS' },
-          { label: 'Santa Catarina', value: 'SC' },
-          { label: 'São Paulo', value: 'SP' },
+          { label: 'Amazonas', value: 'AM', description: 'optional description' },
```

Comment:
The idea was to include the selectedItem logic to the description example. But, as it would require a refectoring spliting the exemples into classes we can go with this and fix the exemples with another issue.